new/LIT7324QeneIfatara.xml

```diff
@@ -0,0 +1,72 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The type of the qǝne is indicated in the manuscript śāhlǝka, which is not mentioned in the classification by 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>, who instead described qǝne
+                    with three verses as <ref type="authFile" corresp="Mibazhu">Mi-bazḫu-type</ref> or 
+                    <ref type="authFile" corresp="Zaamlakiya">Za-ʾamlākiya-type</ref>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Sahleka"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-03-21">Created entity</change>
+            <change when="2025-03-30" who="CH">Corrections according to reviews of Denis Nosnitsin and Jonas Karlsson.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="EMIP00764"/>, fol. 2r. The end of the poem is indicated with ። .</note>
+                <ab>
+                    ሣህልከ፡
+                    <l n="1">ኢፈጠረ፡ ስመ፡ በላዕለ፡ ስሙ፡</l> 
+                    <l n="2">ወል<sic>ዳ</sic>፡ ሥላሴ፡ መስፍን፡ ዘኢይትሜካህ፡ በአቅ<supplied reason="subaudible" resp="CH">ሙ</supplied>፡</l>
+                    <l n="3">እስመ፡ ብሂለ፡ ዝወልደ፡ ሥላሴ፡ ታሕተ፡ ስም፡ አኮ፡ ሳዕለ፡ ስም፡ ዳዕሙ።</l>
```

Comment:
The person in question here is for sure PRS9978Waldase.
The poem may be translated like this: 
"He did not create a name above his name, 
Walda Śǝllāse, prince, who is not boasting with his measure, 
Because saying this, Walda Sellase, is under the name, but not above the name."
The full sense escapes from me, possibly smth about the phisical feature of the person + smth trinitarian. 
But the poem might me an older one. 
"